app/helpers/api/bapi/getAdditionalServices.js

```diff
@@ -14,6 +15,8 @@ export const getAdditionalServices = async ({ addedCatalogueSolutions, accessTok
     return [];
   }
 
+  sortItems(additionalServices, 'name');
```

Comment:
We seem to have 2 different ways of using sortItems.  getOrder.js assigns the output of sortItems to a variable, where as getCatalogueItems.js updates the variable inside the function, like what has been done here.

I believe it is best to assign the output of the sortItems function to a variable, rather than change the value inside a function, as this can cause confusion.

Would it be worth looking at changing the use of sortItems to be consistent between all files ?

Reply:
done